ResearchUXFactory/SBAInfoManager.h

```diff
@@ -68,7 +68,24 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, readonly, copy) NSString * _Nullable keychainAccessGroup;
 
 /**
- App group identifier used for the suite name of NSUserDefaults (if provided).
+ Whether to use standard user defaults as the fallback.
+ For backward compatibility only, when upgrading an app originally built with old
+ Bridge frameworks that used standard user defaults. Has no effect if either userDefaultsSuite
+ or appGroupIdentifier is set.
+ 
+ If neither of those is set, and this is not set or is set to NO, then the default internal suite
+ for Bridge frameworks will be used (org.sagebase.Bridge).
```

Comment:
That's if it's meant for an app group, which this isn't (we don't have an app with the identifier org.sagebase.Bridge). It seemed like following that convention here might be confusing.

And yes, BridgeSDK doesn't look at this when figuring out where to put files (only looks at appGroupIdentifier for that) so it won't cause it to crash; the documentation of NSUserDefaults implies that it would store the info in your local sandbox under the given suite name; and I checked (by deleting appGroupIdentifier from BridgeInfo) that it correctly creates an NSUserDefaults object that isn't the same as the standardUserDefaults one, using this name.